rpcs3/rpcs3qt/settings_dialog.ui

```diff
@@ -3783,6 +3790,7 @@
  </widget>
  <resources>
   <include location="../resources.qrc"/>
+  <include location="../resources.qrc"/>
```

Comment:
Remove this please

Reply:
Idk why this line has been added, I will remove it

Reply:
you might as well remove the size changes above, but it's not really important